From 35dd52a0933be47ba01d865a89bc2a9352640702 Mon Sep 17 00:00:00 2001
From: ArenM <aren@peacevolution.org>
Date: Sat, 23 Apr 2022 13:28:59 -0400
Subject: [PATCH sxmo-utils 3/9] Use KillMode=process so systemd doesn't stop
 apps

Systemd kills processes by cgroup, not session, so the setsid calls in
the inputhandler don't prevent this.
---
 configs/superd/services/sxmo_hook_lisgd.service | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/superd/services/sxmo_hook_lisgd.service b/configs/superd/services/sxmo_hook_lisgd.service
index 182d995..c671235 100644
--- a/configs/superd/services/sxmo_hook_lisgd.service
+++ b/configs/superd/services/sxmo_hook_lisgd.service
@@ -5,3 +5,4 @@ Description=run user defined lisgd start hook
 Type=simple
 Restart=always
 ExecStart=sh -c sxmo_hook_lisgdstart.sh
+KillMode=process
-- 
2.39.2

